app/controllers/charges_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ChargesController < ApplicationController
+  # skip_before_action :authenticate_user!
+
```

Comment:
By doing this you may have introduce bugs! Some charges may require a `current_user`. You'll need to find a different workaround if the anonymous donations are using this